src/App.js

```diff
@@ -1,24 +1,26 @@
-import logo from './logo.svg';
+import { Provider } from 'react-redux';
+import {
+  Route, RouterProvider, createBrowserRouter, createRoutesFromElements,
+} from 'react-router-dom';
+import Home from './routes/Home';
+import Layout from './routes/Layout';
+import Details from './routes/Details';
 import './App.css';
+import store from './redux/store';
 
+const router = createBrowserRouter(
+  createRoutesFromElements(
+    <Route errorElement={<div>No page found</div>} path="/" element={<Layout />}>
+      <Route element={<Home />} path="/" />
```

Comment:
- [ ] Please make sure that the list of countries appears in two columns (2 each row) on the mobile view (375px) as you showed in the video. In the app run and the live demo, it's only one column. kindly fix this to better match the given design.

![image](https://github.com/Mahamadou-Nouridine/metrics-webapp/assets/62290545/dbadca62-4048-4240-9eca-f543b2dcd5c8)